x-pack/legacy/plugins/infra/public/apps/start_app.tsx

```diff
@@ -23,9 +23,12 @@ import { PageRouter } from '../routes';
 import { createStore } from '../store';
 import { ApolloClientContext } from '../utils/apollo_context';
 import { HistoryContext } from '../utils/history_context';
-import { createContext, useUiSetting$ } from '../../../../../../src/plugins/kibana_react/public';
+import {
+  useUiSetting$,
+  KibanaContextProvider,
+} from '../../../../../../src/plugins/kibana_react/public';
 
-const { Provider: KibanaProvider } = createContext(npStart.core);
+const { uiSettings } = npStart.core;
```

Comment:
Can we use this instead of `useUiSetting$` now?

Reply:
Yes, you can use it for few months already. But you cannot statically import `core` in the New Platform. You will need to get it from plugin life-cycle and pass through React context.

Reply:
Yes, you can use it for few months now. But you cannot statically import `core` in the New Platform. You will need to get it from plugin life-cycle and pass through React context.